### DISCUSSION TITLE
增加 Token认证拦截器  typescript demo

### DISCUSSION THREAD

**Discussion by gopower** [alovajs/alova#502] [Q&A]
看了官网文档,  尝试不成功 登录拦截器中保存登录信息 
![image](https://github.com/user-attachments/assets/29f27e4a-b9db-48c6-b775-7a8af521e6ee)

**Comment by MeetinaXD** [comment]
https://github.com/alovajs/alova/blob/main/examples/react/src/views/ClientBased.jsx

**Reply by gopower** [reply]
非常感谢

**Reply by gopower** [reply]
![image](https://github.com/user-attachments/assets/869d6cda-a933-4160-a961-386daca837a7)

**Reply by gopower** [reply]
![image](https://github.com/user-attachments/assets/04a2f47d-61de-4bd7-9ca6-8c11d5c1b9dd)

**Reply by gopower** [reply]
不知道为什么拦截器 里面没有打印

**Comment by MeetinaXD** [comment]
官网有提供React、Vue 和 Svelte 的示例，都包括了 Client 和 Server 的 Token 认证。

如果没找到，这里是地址：[ServerBased.vue](https://codesandbox.io/p/devbox/hardcore-smoke-qqk2vk?file=%2Fsrc%2Fviews%2FServerBased.vue)，这里包含了一个可用的示例

请尝试 clone 仓库在本地使用，如果发现这是个 Bug，请提 issue。
